src/test/java/order/CoffeeOrderBoardTest.java

```diff
@@ -0,0 +1,68 @@
+package order;
+
+import coffee.order.CoffeeOrderBoard;
+import coffee.order.Order;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class CoffeeOrderBoardTest {
+
+    private CoffeeOrderBoard coffeeOrderBoard;
+    private Logger mockLogger;
+
+    public CoffeeOrderBoardTest(Logger mockLogger) {
+        this.mockLogger = mockLogger;
+    }
+
+    @BeforeEach
+    public void setUp() {
+        coffeeOrderBoard = new CoffeeOrderBoard();
+    }
+
+    @Test
+    public void testAddOrder() {
+        coffeeOrderBoard.addOrder("John Doe");
+        List<Order> orders = coffeeOrderBoard.getOrders();
+        assertEquals(1, orders.size());
+        assertEquals(1, orders.get(0).getNumber());
+        assertEquals("John Doe", orders.get(0).getName());
+        verify(mockLogger).log(Level.INFO, "Order added: " + orders.get(0).toString());
```

Comment:
Usually, we don't check how loggers are written. But there - it's good to dive deep into "verify"